src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravResponseDTO.kt

```diff
@@ -21,4 +21,11 @@ data class AktivitetskravVurderingResponseDTO(
     val beskrivelse: String?,
     val arsaker: List<VurderingArsak>,
     val frist: LocalDate?,
+    val varsel: VarselResponseDTO?
+)
+
+data class VarselResponseDTO(
+    val uuid: String,
+    val createdAt: LocalDateTime,
+    // TODO: svarfrist
```

Comment:
Kanskje det går an å legge til en midlertidig defaultverdi med en gang (basert på vurderingens createdAt eller noe)?

Reply:
Jo, vi kunne for så vidt gjort det samme her som vi gjør når varselet opprettes i databasen inntil svarFrist i domene-laget er klart...